.github/workflows/sshnp_docker_demo_base.yml

```diff
@@ -0,0 +1,39 @@
+name: VE_base
+on:
+  workflow_dispatch:
+  schedule:
+    # - cron: '0 0 * * *' # every day, 00:00
+    # - cron: '0 0 * * 0' # every week, 00:00 Sunday
+    - cron: '5 * * * *' # every 5 minutes, temporarily set to 5 minutes for testing
```

Comment:
workflow_dispatch is what we use for testing. This needs to be a sensible weekly parameter before we can merge, or we'll thrash Actions and Docker Hub